packages/scandipwa/src/store/MyAccount/MyAccount.dispatcher.js

```diff
@@ -203,6 +203,52 @@ export class MyAccountDispatcher {
         );
     }
 
+    /**
+     * Resend confirmation email
+     * @param {{email: String}} [options={}]
+     * @memberof MyAccountDispatcher
+     */
+    resendConfirmation(options = {}, dispatch) {
+        const mutation = MyAccountQuery.getResendConfirmationMutation(options);
+
+        return fetchMutation(mutation).then(
+            /** @namespace Store/MyAccount/Dispatcher/MyAccountDispatcher/resendConfirmation/fetchMutation/then */
+            (data) => {
+                const { resendConfirmationEmail: { status = '' } } = data;
+
+                switch (status) {
+                case 'account_confirmation_not_required':
+                    dispatch(showNotification('success', __('This email does not require confirmation.')));
+                    history.push('/customer/account/login');
```

Comment:
If it's possible, please put urls in shareable constants.
Here and in the rest of the code.
This will allow us to reuse constants and reduce issues when we make a type in the constant string values and everything breaks.